installer/default_platform.conf

```diff
@@ -359,6 +359,68 @@ demo_install_uefi_grub()
 
 }
 
+# Install UEFI BIOS SHIM for DEMO OS
+demo_install_uefi_shim()
+{
+    echo "demo_install_uefi_shim(): Installing Secure Boot components"
+
+    local demo_mnt="$1"
+    local blk_dev="$2"
+
+    # make sure /boot/efi is mounted
+    if ! mount | grep -q "/boot/efi"; then
+        mount /boot/efi
```

Comment:
>mount [](http://example.com/codeflow?start=8&length=5)

Need to check the return code.

Reply:
actually, I copied those few lines from a SONiC code in the function `demo_install_uefi_grub` that was coded 7 years ago.
I will add your comment and create a function to avoid code duplication.